.gitignore

```diff
@@ -6,6 +6,7 @@ frontend/node_modules
 beancount_import/frontend_dist/*.js
 beancount_import/frontend_dist/*.css
 beancount_import/frontend_dist/*.map
+beancount_import/frontend_dist/dev
```

Comment:
I think this addition should not be necessary anymore?

Reply:
Yes good call, removed it and added `.eggs/`